plugins/excel-adapter/src/main/java/org/polypheny/db/adapter/excel/ExcelFieldType.java

```diff
@@ -22,14 +22,15 @@
 import org.apache.poi.ss.usermodel.Cell;
 import org.polypheny.db.adapter.java.JavaTypeFactory;
 import org.polypheny.db.algebra.type.AlgDataType;
+import org.polypheny.db.catalog.exceptions.GenericRuntimeException;
 import org.polypheny.db.type.PolyType;
 
 /**
  * Type of a field in a Excel file.
  * <p>
  * Usually, and unless specified explicitly in the header row, a field is of type {@link #STRING}. But specifying the field type in the header row makes it easier to write SQL.
  */
-enum ExcelFieldType {
+public enum ExcelFieldType {
     STRING( String.class, "string" ),
     BOOLEAN( Primitive.BOOLEAN ),
     BYTE( Primitive.BYTE ),
```

Comment:
Same question as with CSV: Why not directly use the new poly type wrappers

Reply:
https://github.com/polypheny/Polypheny-DB/pull/464#discussion_r1521473945